rosidl_generator_c/include/rosidl_generator_c/service_type_support.h

```diff
@@ -0,0 +1,41 @@
+/* Copyright 2014 Open Source Robotics Foundation, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef ROSIDL_GENERATOR_C_ROSIDL_GENERATOR_C_SERVICE_TYPE_SUPPORT_H_
+#define ROSIDL_GENERATOR_C_ROSIDL_GENERATOR_C_SERVICE_TYPE_SUPPORT_H_
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+typedef struct rosidl_service_type_support_t
+{
+  const char * typesupport_identifier;
+  const void * data;
+} rosidl_service_type_support_t;
+
+/* This macro is used to create the symbol of the get_type_support function
+ * for a specific service type. The library of the message package which
+ * defines a given service will provide the symbol to which this macro expands.
+ */
+//#define ROSIDL_GET_SERVICE_TYPE_SUPPORT(SrvPkgName, SrvName) \
+//rosidl_get_service_type_support__##SrvPkgName##__##SrvName()
```

Comment:
Let's add a `TODO` here so we don't forget about it or forget why it is there. Also I've been trying to put the GitHub name of the person who creates the `TODO` in the todo line, e.g. `TODO(esteve): Uncomment this when services are supported in C`.